third_party/amd/include/Dialect/TritonAMDGPU/IR/TritonAMDGPUOps.td

```diff
@@ -32,6 +32,7 @@ include "triton/Dialect/Triton/IR/TritonAttrDefs.td"
 include "triton/Dialect/Triton/IR/TritonInterfaces.td"
 include "triton/Dialect/Triton/IR/TritonOpInterfaces.td"
 include "triton/Dialect/TritonGPU/IR/TritonGPUTypes.td"
+include "triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td"
```

Comment:
I don't think we need to include this given not explicitly using anything from it with the newly added code?